client/src/App.jsx

```diff
@@ -3,8 +3,8 @@ import { Routes, Route } from "react-router-dom";
 import IndexPage from "./pages/IndexPage.jsx";
 import AboutPage from "./pages/About.jsx";
 import SingupPage from "./pages/Singup.jsx";
-import ContactPagePage from "./pages/Contact-page.jsx";
-import QuizMenuPage from "./pages/Quiz-menu.jsx";
+import Contact from "./pages/Contact.jsx";
+import QuizMenuPage from "./pages/QuizMenu.jsx";
```

Comment:
no hace falta escribir las extensiones si son archivos javascript/typescript
it could be : `import QuizMenuPage from "./pages/QuizMenu";`

Reply:
Deleted extension from file names